src/views/v2/rsvp/details/availability/days-to-rsvp.php

```diff
@@ -17,25 +17,43 @@
  * @version TBD
  */
 
-// Bail if RSVP isn't in stock.
-if ( ! $rsvp->is_in_stock() ) {
+use Tribe__Date_Utils as Dates;
+
+$days_to_rsvp = Dates::date_diff( current_time( 'mysql' ), $rsvp->end_date );
+$days_to_rsvp = floor( $days_to_rsvp );
+
+// Only show this if it is happening within the next week.
+if ( 6 < $days_to_rsvp ){
```

Comment:
* :no_entry_sign: Expected 1 space(s) after closing parenthesis; found 0
* :no_entry_sign: You must have a space after a closing parenthesis
* :no_entry_sign: Space between opening control structure and closing parenthesis is required

<sub>hash: 3042137258f42a567c464de8977492b4</sub>